app/src/main/cpp/skyline/services/ldn/IUserLocalCommunicationService.cpp

```diff
@@ -9,8 +9,24 @@ namespace skyline::service::ldn {
           event{std::make_shared<type::KEvent>(state, false)} {}
 
     Result IUserLocalCommunicationService::GetState(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
-        static constexpr u32 StateNone{0x0};
-        response.Push<u32>(StateNone);
+        static constexpr u32 StateError{0x6};
+        response.Push<u32>(StateError);
+        return {};
+    }
+
+    Result IUserLocalCommunicationService::GetNetworkInfo(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
```

Comment:
Please implement the below funcs properly, at least returning fake data

Reply:
@bylaws I already have something stubbed for this. I can implement these ldn funcs to return correct data, is this good idea?